packages/features/webhooks/pages/webhook-new-view.tsx

```
@@ -84,7 +84,8 @@ export const NewWebhookView = ({ webhooks, installedApps }: Props) => {
       <WebhookForm
         noRoutingFormTriggers={false}
         onSubmit={onCreateWebhook}
-        apps={installedApps?.items.map((app) => app.slug)}
+        apps={[...installedApps?.items.map((app) => app.slug), "cal-video"]}
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)